README.md

```diff
@@ -155,6 +155,12 @@ You can deploy a static build of our Storybook site to gh-pages from the master
 1. First be sure you've built a static version of Storybook: `yarn build:storybook`
 2. Then deploy to gh-pages: `yarn release:site`
 
+### To deploy to your own mirror of leafygreen-ui
```

Comment:
Does this assume you've already created your fork?

Reply:
Oh, yeah it does. I need to think about how to approach that case.